engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -91,6 +91,23 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
       }
     }
 
+  def submitBatchRoute =
+    path("workflows" / Segment / "batch") { version =>
```

Comment:
Thinking out loud: Is there a reason for making this a separate endpoint? I would think the existing submit could be modified to allow either single or multiple inputs?